cli/auth.go

```diff
@@ -183,7 +226,7 @@ func (c *viamClient) logoutAction(cCtx *cli.Context) error {
 	if err := c.logout(); err != nil {
 		return errors.Wrap(err, "could not logout")
 	}
-	fmt.Fprintf(cCtx.App.Writer, "logged out from %q\n", auth.User.Email)
+	fmt.Fprintf(cCtx.App.Writer, "logged out from %s\n", auth)
```

Comment:
[minor / ignorable if it's going to take more than a few min] It would be nice if the error messages and flow of the code were more similar for API keys and users. For example they both have the same set of messages:
```
was logged in with an [api-key|user account] with ID %s. logging out.
```
and 
```
%slogged in as %q, expires %s\n
```
(where expires is not set for API key I think)

Reply:
I could jam the two of them together but the code paths intuitively look different to me. They create the auth method differently, have different rules for overwriting existing auth, and verify the auth method differently. If it would help, I can make the log messages more similar. But combining into one code path is not obviously cleaner to me (If I am missing some obvious clean way to do this, please lmk!)